src/automaton.ts

```diff
@@ -247,6 +250,20 @@ export class State {
     return eqSet(this.set, set)
   }
 
+  // Only use after all mutations to this.actions are complete
+  filterByTerm(term: Term) {
+    if (this.actionsMapCache == null) {
```

Comment:
It'd be nice if you tried to use spacing consistent with the rest of the file (I see 3 styles for `for`/`if` in this function, and no spaces around the `=` below).

Reply:
fixed